mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -64,6 +69,8 @@ public enum SymbolScaling {ALL, POI}
      */
     public static int NUMBER_OF_THREADS = Runtime.getRuntime().availableProcessors() + 1;
 
+    public static final long MAX_MEMORY_MB = Runtime.getRuntime().maxMemory() / 1000 / 1000;
```

Comment:
@Sublimis This is not a parameter, but a calculated constant. It should be moved to the class that uses it.